### ISSUE TITLE
Show strings with quote marks around them?

### ISSUE THREAD

**Issue by danielcompton** [binaryage/cljs-devtools#17]
I was helping debug a [question about re-frame](http://stackoverflow.com/questions/37751108/re-frame-subscription-not-working-as-expected/37779552). The root cause was that a parameter was a string instead of an integer.

The devtools output for calling the function with a string and a number are quite similar, the only difference is slight syntax highlighting. When `id` is a string (first image), it is darker coloured, but isn't wrapped in quotes. In the second one it is blue. Perhaps this is just the way devtools works, and a fix isn't easy/possible here, but I wonder if wrapping it in quotes could be helpful?

![pasted_image_at_2016_06_12_11_11_pm_720](https://cloud.githubusercontent.com/assets/811954/15994128/99f75c5e-314f-11e6-9995-5d3bd8d1d30b.png)
![pasted_image_at_2016_06_12_11_03_pm_720](https://cloud.githubusercontent.com/assets/811954/15994130/a63c1464-314f-11e6-976d-99eda0888a98.png)

**Comment by darwin** [comment]
It could be possible, but I'm afraid I'm not going to override standard formatters for strings. That feels too intrusive. I'm afraid we could open a can of worms by trying to take over DevTools's job of formatting raw JS strings.

**Comment by darwin** [comment]
I could implement a helper function which would "mark" a raw javascript thing as something to be printed by cljs-devtool's formatter. So you would explicitly ask for it to be rendered in cljs-devtools-way.

Similar to envelope: https://github.com/binaryage/cljs-devtools/blob/master/src/lib/devtools/toolbox.cljs#L5

**Comment by danielcompton** [comment]
Hmm, not sure if the cure would be worse than the disease.

**Comment by darwin** [comment]
I just did some quick tests and DevTools do not call custom formatters for primitive values like numbers, nulls or strings. I get a chance to format something only if it is wrapped in some non-primitive value (javascript object).

Also I noticed pretty weird behaviour of console formatting for strings. If the first argument to console.log is formatting string, then all following strings are black without quotes (they get rendered by string formatting routine). If the first argument is something else then strings are red with black quotes around them (they get rendered as objects with their internal string formatter).

I think it can be explained by two different rendering paths taken with formatting string and without it. But it is confusing at least.
